auth-web/src/util/constants.ts

```diff
@@ -59,6 +59,7 @@ export enum Role {
     ViewAllTransactions = 'view_all_transactions',
     ManageEft = 'manage_eft',
     EftRefund = 'eft_refund',
+    EftRefundApprover = 'eft_refund_approver',
```

Comment:
should we make this similar to the business language? `expensive_authority` role? thoughts? @ochiu @Jxio